.github/workflows/linux_arm64.yml

```diff
@@ -140,6 +145,7 @@ jobs:
           pip-cache-path: ${{ env.PIP_CACHE_PATH }}
           should-setup-pip-paths: 'true'
           show-cache-info: 'true'
+          skip-python-install: 'true'
```

Comment:
should we just remove it?

Reply:
This action also sets up pip cache paths, that's why I kept it. Alternatively, I can hardcode the path to pip cache right in the Dockerfile, but not sure if it's convenient (@akashchi, how do you think?)

Or I can create a separate action for setting up pip caches

Reply:
This action should skip python installation if it is already installed. why do we need to manually skip it? 
Cache can be set without it (by setting  PIP_CACHE_DIR env)

Reply:
We need to skip it to avoid calling apt - I'm not sure if it won't try to connect to problematic launchpad servers even if the dependency is already installed (like with apt cache)

Reply:
then we need to remove that step at all, it is already too complicated